locales/tr-TR.js

```diff
@@ -0,0 +1,8 @@
+export const val = {
+	'buttonCancel': 'İptal',
+	'buttonUpdate': 'Güncelle',
+	'inputLabelPoints': 'Puanlar',
+	'mainBodyDescription': 'Buradaki soruların detayları güncellendiğinde Brightspace'teki sınava uygulanır ancak Harici Eğitim aracında oluşturulan asıl soruyu etkilemez.',
```

Comment:
Need to escape single quotes
```suggestion
	'mainBodyDescription': 'Buradaki soruların detayları güncellendiğinde Brightspace\'teki sınava uygulanır ancak Harici Eğitim aracında oluşturulan asıl soruyu etkilemez.',
```